sdk/pr.yml

```diff
@@ -0,0 +1,23 @@
+trigger:
+  branches:
+    exclude:
+    - pipelinev3-v2
```

Comment:
Should the trigger and PR refs change now?

Reply:
Yes.